test/functional/wintps/build.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0"?>
+
+<!--
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+-->
+
+<project name="WinTPS Installer tests" default="build" basedir=".">
+	<taskdef resource="net/sf/antcontrib/antlib.xml" />
+	<description>
+		WinTPS Installer tests
+	</description>
+
+	<!-- set global properties for this build -->
+	<property name="DEST" value="${BUILD_ROOT}/functional/wintps" />
+	
+	<!--Properties for this particular build-->
+	<property name="src" location="./" />
+	
+	<target name="wintpsTest.check">
+		<condition property="wintpsTestdir.exists">
+			<available file="WinTPSTest" type="dir"/>
+		</condition>
+	</target>
+
+	<target name="getWinTPSTest" depends="wintpsTest.check" unless="wintpsTestdir.exists">
+		<exec executable="git" failonerror="true">
+			<arg value="clone" />
+			<arg value="https://github.com/RadekCap/WindowsTPS" />
```

Comment:
Are we happy with the source of this lib?

Reply:
Good catch, that should now be updated to https://github.com/rh-openjdk/WindowsTPS